CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.1.0 (2015-09-02)
```

Comment:
If we use `.rst` files for the README and CHANGELOG, then we can easily use them for the `long_description` on PyPI and it'll look pretty, which is were most people will look at your library anyway.

[Related Demands issue](https://github.com/yola/demands/issues/30)

Though we don't actually use it for the `long_description` on PyPI  yet there either, though here's an example in a cookiecutter: https://github.com/audreyr/cookiecutter-pypackage/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/setup.py#L11-L29